pkg/utils/utils.go

```diff
@@ -39,3 +44,33 @@ func Contains(s []string, str string) bool {
 	}
 	return false
 }
+
+func SetEnv(envName string) error {
```

Comment:
we can split this function into 2 functions. 1 to read/write data into a file and the other to look for a regex in string.